tests/sentry/issues/endpoints/test_organization_group_index.py

```diff
@@ -753,6 +753,38 @@ def test_lookup_by_release(self, _: MagicMock) -> None:
         assert len(issues) == 1
         assert int(issues[0]["id"]) == event.group.id
 
+    def test_release_package_in(self, _: MagicMock) -> None:
+        self.login_as(self.user)
+        project = self.project
+        release1 = Release.objects.create(organization=project.organization, version="foo@1.0.0.0")
+        release2 = Release.objects.create(organization=project.organization, version="bar@1.2.0.0")
+        release1.add_project(project)
+        release2.add_project(project)
+
+        event1 = self.store_event(
+            data={
+                "release": release1.version,
+                "timestamp": iso_format(before_now(seconds=3)),
+                "fingerprint": ["1"],
+            },
+            project_id=project.id,
+        )
+        event2 = self.store_event(
+            data={
+                "release": release2.version,
+                "timestamp": iso_format(before_now(seconds=2)),
+                "fingerprint": ["2"],
+            },
+            project_id=project.id,
+        )
```

Comment:
Worth adding an event that isn't in either release to be sure this is filtering correctly?